datahub-web-react/src/graphql/search.graphql

```diff
@@ -262,12 +262,34 @@ fragment autoCompleteFields on Entity {
         dataPlatformInstance {
             ...dataPlatformInstanceFields
         }
+        featureTables: relationships(input: { types: ["Contains"], direction: INCOMING, start: 0, count: 1 }) {
+            relationships {
+                entity {
+                    ... on MLFeatureTable {
+                        platform {
+                            ...platformFields
+                        }
+                    }
+                }
+            }
+        }
```

Comment:
oh wow is this what we do for ml feature and ml primary key to get their platforms? that's nuts lol